### PR TITLE
Renamed dcc-genome db

### PR DESCRIPTION
**Pull request by slovit**

### CODE REVIEW

File: dcc-etl-core/src/test/conf/config.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ identifierServiceUri: "http://localhost:6381"
 projectMongoUri: "mongodb://localhost/dcc-project"
```
Comment (btiernay) [inline dcc-etl-core/src/test/conf/config.yaml:50]: This used any more? I don't think so...

File: dcc-etl-client/src/test/conf/config.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ identifierServiceUri: "http://localhost:6381"
 projectMongoUri: "mongodb://localhost/dcc-project"
 
 # Gene MongoDB URI
-geneMongoUri: "mongodb://localhost/dcc-genome"
+geneMongoUri: "mongodb://localhost/dcc-import"
 
 # CGHub BSON dump
 cghubFile: "target/cghub.bson"
```
Comment (btiernay) [inline dcc-etl-client/src/test/conf/config.yaml:56]: Same...

File: dcc-etl-client/src/test/conf/config.yaml [hunk 3]
```diff
@@ -50,7 +50,7 @@ identifierServiceUri: "http://localhost:6381"
 projectMongoUri: "mongodb://localhost/dcc-project"
```
Comment (btiernay) [inline dcc-etl-client/src/test/conf/config.yaml:50]: Same...

File: dcc-etl-core/src/test/conf/config.yaml (hunk 1 above)
Comment (slovit) [inline dcc-etl-core/src/test/conf/config.yaml:50]: Seems not to be used.

**Comment by btiernay** [comment]
@slovit Seems to be broken. Can you please have a look? Thanks

**Comment by slovit** [comment]
It's not. GitHub failed to pickup results of a manual build (http://dcc-jenkins1.oicr.on.ca:8080/job/dcc-etl-pr/214/)

**Pull request merged**